src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2282,15 +2300,15 @@
     <file>env_run.xml</file>
     <desc>pio root processor relative to component root</desc>
     <values>
-      <value compclass="ATM">1</value>
-      <value compclass="CPL">1</value>
-      <value compclass="OCN">1</value>
-      <value compclass="WAV">1</value>
-      <value compclass="GLC">1</value>
-      <value compclass="ICE">1</value>
-      <value compclass="ROF">1</value>
-      <value compclass="LND">1</value>
-      <value compclass="ESP">1</value>
+      <value component="ATM">0</value>
```

Comment:
Why this change?   Should be made in an acme specific way such as in config/machines/config_pio.xml

Reply:
@jayeshkrishna ?

Reply:
I will put this in config_pio.xml, do it for all machines.